Trigger/python/genTriggerFcl.py

```diff
@@ -240,7 +240,7 @@ def generate(configFileText="allPaths", verbose=True, doWrite=True):
     #
 
     configFile = open(configFileName, "r")
-
+    pathID     = 100 #we start the Reco paths from 100
```

Comment:
How many other pieces of code need to know that 100 is a magic number?   I see that this is in the python world.  Will we need a place in the C++ world or the tuple world?  How can we define it once in each world?

Reply:
That is something that was discussed a long time ago in some software meeting; I proposed to leave free the first 100 bits for non-triggger paths. No other code needs to know this number. This script generates only the trigger Menu file. An analyzer will need to know either the list of the "good" bits or the list of the pathnames.